providers/implementations/digests/build.info

```diff
@@ -15,7 +15,7 @@ $MD2_GOAL=../../liblegacy.a
 $MD4_GOAL=../../liblegacy.a
 $MDC2_GOAL=../../liblegacy.a
 $WHIRLPOOL_GOAL=../../liblegacy.a
-$RIPEMD_GOAL=../../liblegacy.a
+$RIPEMD_GOAL=$COMMON_GOAL
```

Comment:
Instead of libcommon, this should go to both liblegacy and libdefault.

Reply:
Doing this causes build problems: https://github.com/openssl/openssl/actions/runs/3224580400/jobs/5275865470